app/views/projects/index.html.erb

```diff
@@ -32,6 +34,11 @@
     <% @projects.each do |project| %>
       <tr>
         <td><%= project.id %></td>
+        <td>
+          <% if project.halted? %>
+            <i class="glyphicon glyphicon-stop text-muted"></i>
+          <% end %>
```

Comment:
not blocking at all - maybe add a green tick or 'ok' if it's not halted etc.

Reply:
since the vast majority will be active, we don't want to clutter the ui with icons for it